OLS Release Notes 1.1.2 further updates (#117406)
openshift/openshift-docs: modules/ols-about-the-offline-knowledge-portal.adoc

OKP requires approximately 75 GiB of ephemeral storage for Solr data, along with 2 CPU and 2 GiB of memory by default. Cluster administrators can tune these resource allocations by using `spec.ols.deployment.rhokp` on the `OLSConfig` custom resource (CR).

[IMPORTANT]
====

> The initial setup of the OKP RAG system might take some time while the documentation corpus is indexed. During this period, {ols-long} might not return documentation-grounded answers. Wait for the indexing process to complete before expecting full RAG functionality.

Author: ```suggestion
The initial setup of the OKP RAG system might take some time during the pod installation. During this period, {ols-long} might not be available. Wait for the indexing process to complete before expecting full RAG functionality.
```
